qt-models/divetripmodel.cpp

```diff
@@ -157,7 +157,8 @@ QVariant DiveItem::data(int column, int role) const
 			retVal = displayDate();
 			break;
 		case DEPTH:
-			retVal = displayDepth();
+			if (prefs.units.show_units_table) retVal = displayDepthWithUnit();
+			else retVal = displayDepth();
```

Comment:
~~could you please use separate lines for the `if / then` branching.
e.g. `retVal = displayDepthWithUnit()` and `retVal = displayDepth();` on new lines.~~

Reply:
...or just write it as:
```
retVal = prefs.units.show_units_table ? displayDepthWithUnit() : displayDepth();
```